### DISCUSSION TITLE
Change spacing after second level dot point when followed by another first level dot point

### DISCUSSION THREAD

**Discussion by RohanAlexander** [quarto-dev/quarto-cli#4154] [Q&A] [answered]
Does anyone know how to adjust the spacing after a second level dot point when it is followed by another first level dot point?
Example of current behaviour:
<img width="683" alt="Screen Shot 2023-01-30 at 9 06 22 PM" src="https://user-images.githubusercontent.com/5662323/215642023-c36a1485-bd69-482d-815f-80a3e0f1248d.png">

Mock up of desired behaviour:
<img width="683" alt="Screen Shot 2023-01-30 at 9 06 22 PM" src="https://user-images.githubusercontent.com/5662323/215642295-bcd98ccb-9fac-4e0a-9e25-685998d0592b.png">

**Comment by baptiste** [comment, marked as answer]
maybe something like this

````
---
title: "Untitled"
format:  
  html:
    include-in-header: 
      - text: |
          <style>
          ul li ul {
           margin-bottom:0em;
           }
          </style>
---

- item
  - subitem
  - subitem
  - subitem
- item
- item
  - subitem
- item

text
````

**Reply by RohanAlexander** [reply]
This works perfectly. Thank you very much.
